tools/mlir-proto-opt/mlir-proto-opt.cpp

```diff
@@ -10,9 +10,9 @@
 //
 //===----------------------------------------------------------------------===//
 
-#include "./lib/CAPI.h"
-#include "include/LinalgExt/LinalgExtDialect.h"
-#include "include/LinalgExt/Passes.h"
+#include "CAPI.h"
+//#include "include/LinalgExt/LinalgExtDialect.h"
```

Comment:
QQ as to why this is needed ?
Not a blocker, we can iterating after landing this.

Reply:
It was missing build support so I commented it out. That has now been resolved, so re-enabled.